### PR TITLE
The Heat Maps concepts 

### LINKED ISSUE #5741: Getting started with heatMaps in Angular 12
#### Proposed title of article
Getting started with heatMaps in Angular 12
#### Proposed article introduction
In the article, I would like to describe a way to easily implement a simple heat map in an Angular application. For that, I will use two external libraries, which are going to be Leaflet.js and Heatmaps.js. Before the implementation of the heat map, I prefer to begin with a brief introduction to the libraries I mentioned above. Leaflet.js It is one of the most popular JavaScript libraries for interactive maps. It has over 30k stars on GitHub. There is an alternative to it such as OpenLayers. But in my example, I picked a leaflet.

#### Key takeaways 
- intro to leaflet.js
- working with heatmaps
- Module injection

### PR DESCRIPTION
**Pull request by NaomiSeint**
## Our **Peer Reviewer and Content Moderator teams do NOT provide any revisions services.** All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner. 

**ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.**
#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

#### Topic suggestion this closes
This closes #5741

### CODE REVIEW

File: content/articles/angular-heat-maps/index.md [hunk 1]
```diff
@@ -0,0 +1,447 @@
...
+
+@Component({
+selector: 'my-app',
+template:
+   `<ejs-heatmap id='container' style="display:block;" [paletteSettings]='paletteSettings'
+   [cellSettings]='cellSettings' [titleSettings]='titleSettings' [dataSource]='dataSource'
+   [xAxis]='xAxis' [yAxis]='yAxis' [legendSettings]='legendSettings'>
+    </ejs-heatmap>`,
+encapsulation: ViewEncapsulation.None
+})
```
Comment (jumamiller) [inline content/articles/angular-heat-maps/index.md:322]: Hi, kindly explain this part well, like give readers with a detailed explanation on what the code does.

File: content/articles/angular-heat-maps/index.md [hunk 2]
```diff
@@ -0,0 +1,447 @@
...
+This tutorial will guide you through the steps needed to make a heatmap and exhibit the elementary usage of heatmap control.
+
+### Table of content
+-[Introduction](#Introduction)
+-[Table of content](#Table-of-content)
+-[Prerequisite](#Prerequisite)
+-[Objective](#objective)
+-[Setting up the project](#Setting-up-the-project)
+
+### Prerequisite
```
Comment (jumamiller) [inline content/articles/angular-heat-maps/index.md:15]: Fix this line, should be `prerequisites`.

File: content/articles/angular-heat-maps/index.md [hunk 3]
```diff
@@ -0,0 +1,447 @@
...
+To be guided by this tutorial, you'll need the following;
+-basic knowledge in Node.js
+-Basic knowledge in typescript and Angular
+-Angular CLI locally installed
+-Local development environment
+
+### Objective
+At the end of this tutorial, you will be capable of building a reactive web applications using heat maps concept.
```
Comment (jumamiller) [inline content/articles/angular-heat-maps/index.md:23]: Could add more contents here?

File: content/articles/angular-heat-maps/index.md [hunk 4]
```diff
@@ -0,0 +1,447 @@
...
+```
+
+### Register the heatmap module
+from the syncfusion  Heatmap package ,import the heatmap module into the angular application(app.module.ts).
+@syncfusion/ej2-ng-heatmap [src/app/app.module.ts].
+
+```typescript
+import { NgModule }      from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+// import the HeatMapModule for the heatmap component
```
Comment (jumamiller) [inline content/articles/angular-heat-maps/index.md:55]: Why is this commented out?

File: content/articles/angular-heat-maps/index.md [hunk 5]
```diff
@@ -0,0 +1,447 @@
...
+       [60, 77, 74, 68, 88, 51],
+       [25, 25, 10, 12, 78, 14],
+       [25, 56, 55, 58, 12, 82],
+       [74, 33, 88, 23, 86, 59]
+    ];
+}
+
+```
+
+In the above code snippet, we export the HeatMapModule from the @syncfusion/ej2-angular-heatmap package and import it into the @NgModule.
```
Comment (jumamiller) [inline content/articles/angular-heat-maps/index.md:148]: Add more content on this part...

**Review by jumamiller — CHANGES_REQUESTED** [review]
Kindly make the requested changes, then we're good!

**Comment by NaomiSeint** [comment]
> jumamiller

Hi, all changes are done!

**Comment by jumamiller** [comment]
> > jumamiller
> 
> Hi, all changes are done!

Great, let's proceed to tech review!

**Comment by louisefindlay23** [comment]
Your article appears to have many sentences that are taken verbatim from third-party sources. We can't publish any content that is not original or properly cited/referenced. There are many free online tools where you can check this (such as quetext.com, and if you're uncertain about the definitions around plagiarism, this is a [good resource to reference](https://www.scribbr.com/category/plagiarism/). We're really looking for unique/original content.

[HeatMap Concepts Copyleaks Report.pdf](https://github.com/section-engineering-education/engineering-education/files/8027270/HeatMap.Concepts.Copyleaks.Report.pdf)
.

**Pull request closed without merging**